.packit.yaml

```diff
@@ -17,3 +17,7 @@ jobs:
     targets:
     - fedora-all
     - epel-8-x86_64
+
+srpm_build_deps:
```

Comment:
This was apparently still missing in the master repo: I'm not 100% certain if this is correct. Please verify.

Reply:
I'm still not sure what this fix is fixing, we had not such issue previously